server/vault.api.js

```diff
@@ -0,0 +1,128 @@
+const express = require('express');
+const asyncHandler = require('express-async-handler');
+const HomebrewModel = require('./homebrew.model.js').model;
+
+const router = express.Router();
+
+const buildTitleConditions = (title) => {
+    if (!title) return {};
+    return {
+        $text: {
+            $search: title,
+            $caseSensitive: false,
+        },
+    };
+};
```

Comment:
Could be slimmed down and remains legible

```suggestion
const buildTitleConditions = (title) => {
    if (!title) return {};
    return {$text: {$search: title, $caseSensitive: false}};
};
```

Reply:
I do prefer it spanning a few lines, like my CSS, if i can choose here